presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeometryType.java

```diff
@@ -59,12 +59,20 @@ public Slice getSlice(Block block, int position)
     @Override
     public void writeSlice(BlockBuilder blockBuilder, Slice value)
     {
+        if (value == null) {
```

Comment:
Is this a bug that you found while adding this new function? If it is, it would be nice to fix this in a separate commit to make it easier to track.